src/utils/index.js

```diff
@@ -0,0 +1,19 @@
+// eslint-disable-next-line
```

Comment:
This override doesn't seem necessary – I imagine it's to circumvent the rule "no solo `const` export in a file". Could this be moved as an `export default` in a separate file instead?